### DISCUSSION TITLE
Form Builder V3: Field Defer

### DISCUSSION THREAD

**Discussion by monointeractive** [filamentphp/filament#8441] [Help]
**The Form Builder V3 documentation says that fields use the defer mode by default:**
_"When you bind a field to a state path, you may use the defer modifier to ensure that the state is only sent to the server when the user submits the form, or whenever the next Livewire request is made. This is the default behaviour."_

Unfortunately, any character entered in the text field sends a request immediately, after each letter.

How to configure fields so that the field status is sent when the form is submitted.

(We're talking about Filament\Forms\Components\ViewField)

**Comment by leandrocfe** [comment]
Can you share the code please?

**Reply by monointeractive** [reply]
```
<x-dynamic-component
    :component="$getFieldWrapperView()"
    :field="$field"
>
    <div x-data="{ state: $wire.{{ $applyStateBindingModifiers("\$entangle('{$getStatePath()}')") }} }">
      <input x-model="state" />
    </div>
</x-dynamic-component>
```

**Reply by leandrocfe** [reply]
```html
<input wire:model="{{ $getStatePath() }}" x-model="state" />
```

**Reply by monointeractive** [reply]
<img width="669" alt="image" src="https://github.com/filamentphp/filament/assets/44318178/f759c23a-c03f-4bc9-abf3-4a34e9271870">

Thanks. However, I wanted to use the second option, recommended in the documentation: $wire.$entangle()
Unfortunately this doesn't work in "defer mode".
